Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -188,14 +159,15 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     func tableView(tableView: UITableView, editActionsForRowAtIndexPath indexPath: NSIndexPath) -> [AnyObject]? {
         if source == nil {
             return [AnyObject]()
-        } else if source!.current.count == 0 && source!.current.guid == BookmarkRoots.MobileFolderGUID {
+        }
+
+        if !(source?.current.itemIsEditableAtIndex(indexPath.row) ?? false) {
```

Comment:
Also this will prevent the default bookmark row from jiggling to the side a bit when theres no delete actions.